### ISSUE TITLE
Command "tenancy:migrate:fresh" is not defined

### ISSUE THREAD

**Issue by pedzed** [hyn/multi-tenant#340]
Since Laravel 5.5, you can not only `migrate:refresh`, but also `migrate:fresh`. This is unfortunately not supported for tenants (`tenancy:migrate:fresh`).

```sh
$ php artisan tenancy:migrate:fresh --seed
```
```
  [Symfony\Component\Console\Exception\CommandNotFoundException]
  Command "tenancy:migrate:fresh" is not defined.

  Did you mean one of these?
      tenancy:migrate
      tenancy:migrate:refresh
      tenancy:migrate:reset
      tenancy:migrate:rollback
```

https://laravel-news.com/migrate-fresh
https://laravel.com/docs/5.5/migrations#running-migrations ("Drop All Tables & Migrate")

**Comment by Thijmen** [comment]
We currently do not support this, feel free to create this with a PR! :)

**Comment by mintunitish** [comment]
I will like to work on this!

**Comment by luceos** [comment]
@mintunitish thanks Nitish, feel free to create a WIP PR early on so we can jump in to offer advice.

**Comment by mintunitish** [comment]
@luceos I have created a WIP PR. here's the link: [PR](https://github.com/hyn/multi-tenant/pull/500).
I have registered the command in the ConnectionProvider, still the command is not being recognized, is there something that I am missing?

**Comment by mintunitish** [comment]
@luceos I think this issue was resolved in #516 and can be closed now.
